google-chat-bot/build.gradle.kts

```diff
@@ -38,28 +42,30 @@ spine {
     enableJava().server()
 }
 
-dependencies {
-    annotationProcessor(enforcedPlatform(Deps.build.micronaut.bom))
-    annotationProcessor(Deps.build.micronaut.injectJava)
-    annotationProcessor(Deps.build.micronaut.validation)
-
-    compileOnly(enforcedPlatform(Deps.build.micronaut.bom))
+micronaut {
+    runtime(MicronautRuntime.NETTY)
```

Comment:
Should we consider doing this with Bootstrap?

Reply:
Configuring the Micronaut runtime? I don't think so. It supports 9 runtimes as of now and for me, it is definitely not Bootstrap's responsibility.